client/boot/index.js

```diff
@@ -1,4 +1,9 @@
 /**
+ * Internal dependencies
+ */
+import { setReduxStore as setSupportUserReduxStore } from 'lib/user/support-user-interop';
```

Comment:
Maybe leave this grouped with the existing internal dependencies, and we can add a separate janitorial PR to update import usages.